extensions/podman/package.json

```diff
@@ -20,6 +20,11 @@
     "configuration": {
       "title": "Podman",
       "properties": {
+        "podman.binary.path": {
+          "type": "string",
+          "default": "",
+          "description": "Custom path to Podman binary. If gvproxy and podman-mac-helper are required, update helper_binaries_dir in containers.conf. (Default is blank)"
```

Comment:
maybe remove the mention to podman-mac-helper and gvproxy as on linux or Windows you don't know what these binaries are

Reply:
Ah! Got it, I'll remove that and add it to documentation instead.